### PR TITLE
fix(#5807): align target cards to column

### LINKED ISSUE #5807: Fix widget wrapping when displaying multiple columns
On larger displays with multiple columns of widgets, if there is an extra one or two widgets at the end they should align left and conform to the columns above, instead of centering.

Screen Shot 2019-07-15 at 12 19 36 PM
When there are three columns of widgets, any widgets in the final row should start in the column on the far left, and fill to the right. In this case, the final widget should be in the first column on the left-hand side.

Screen Shot 2019-07-15 at 12 19 49 PM

When there are two columns of widgets, the current look is even worse! Widgets at the end should not break free of the established columns, as this makes them look special in some way. The last widget should be in the first column on the left-hand side.

### PR DESCRIPTION
**Pull request by Benmuiruri**
# Description

Updates the Targets page to align the `target` card to the columns

<details>
  <summary>Layout with one card in last row</summary>

Screenshot 2024-03-20 at 09 30 23

</details>

<details>
  <summary>Layout with two cards in last row</summary>

Screenshot 2024-03-20 at 09 31 00

</details>

<details>
  <summary>Layout with three cards in last row</summary>

Screenshot 2024-03-20 at 09 32 15

</details>

<details>
  <summary>Mobile view</summary>

Screenshot 2024-03-20 at 09 32 41

</details>

<details>
  <summary>Tablet view</summary>

Screenshot 2024-03-20 at 15 56 40

</details>

<details>
  <summary>Firefox</summary>

Screenshot 2024-03-20 at 15 55 50

</details>

Closes #5807 

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs
If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:5807-align-target-cards/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:5807-align-target-cards/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:5807-align-target-cards/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: webapp/src/css/targets.less [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .targets {
-  display: flex;
-  align-items: stretch;
-  flex-flow: row wrap;
+  display: grid;
+  grid-template-columns: repeat(3, 296px);
+  gap: 0 24px;
```
Comment (latin-panda) [inline webapp/src/css/targets.less:4]: Please extract the 24 in a variable

**Review by latin-panda — CHANGES_REQUESTED** [review]
Please test in 
- firefox
- chrome
- desktop, phone and tablet

**Review by n-orlowski — APPROVED** [review]
LGMT! 🎉

**Review by latin-panda — APPROVED** [review]
Thanks! LGTM!

**Pull request merged**